emmet-api/emmet/api/routes/materials/robocrys/query_operators.py

```diff
@@ -32,43 +32,26 @@ def query(
                         "path": "description",
                         "allowAnalyzedField": True,
                     },
+                    "sort": {"score": {"$meta": "searchScore"}, "description": 1},
+                    "count": {"type": "total"},
                 }
             },
+            {"$skip": _skip},
+            {"$limit": _limit},
             {
-                "$facet": {
-                    "total_doc": [{"$count": "count"}],
-                    "results": [
-                        {
-                            "$project": {
-                                "_id": 0,
-                                "task_id": 1,
-                                "material_id": 1,
-                                "description": 1,
-                                "condensed_structure": 1,
-                                "last_updates": 1,
-                                "search_score": {"$meta": "searchScore"},
-                            }
-                        }
-                    ],
-                }
-            },
-            {"$unwind": "$results"},
-            {"$unwind": "$total_doc"},
-            {
-                "$replaceRoot": {
-                    "newRoot": {
-                        "$mergeObjects": ["$results", {"total_doc": "$total_doc.count"}]
-                    }
+                "$project": {
+                    "_id": 0,
+                    "meta": "$$SEARCH_META",
+                    "material_id": 1,
```

Comment:
Looks like there was a typo in the original pipeline. We should keep `last_updated` as a projected field.